src/main/java/com/github/rmannibucau/oauth2/backend/JPAOAuthDataProvider.java

```diff
@@ -149,6 +154,16 @@ private TokenEntity toEntity(final ServerAccessToken serverToken, final TokenEnt
         serverAccessToken.setExpiresIn(tokenEntity.getExpiresIn());
         serverAccessToken.setIssuedAt(tokenEntity.getIssuedAt());
         serverAccessToken.setIssuer(tokenEntity.getIssuer());
+        
+        // why is this never visible on the bearer token?
+        // there is a parameter section
+        if (tokenEntity.getRoles() != null) {
+			Map<String, String> parameters = new HashMap<>();
+			parameters.put("roles", tokenEntity.getRoles());
+			serverAccessToken.setParameters(parameters);
+			System.out.println("ADDED ROLES");
```

Comment:
do you want to drop this sysout?

side note: seems the formatting is not aligned with the rest of the class, can you fix it please?